build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -242,6 +242,10 @@ LIBRARY CoreGraphics
         _CGContextCreateWithD2DRenderTarget
         _CGGetD2DFactory
         _CGGetPixelFormatProperties
+        _CGContextPushBeginDraw
+        _CGContextPopEndDraw
+        _CGContextEscapeBeginEndDrawStack
```

Comment:
I think BeginDrawIfNeeded(), EndDrawIfNeeded() are more descriptive than EscapeStack.  Won't fix is okay.